src/app/trips/shared/trip.service.ts

```diff
@@ -33,19 +33,19 @@ export class TripService {
   }
 
   // create new trip
-  createTrip(trip: ITrip) {
+  createTrip(trip: ITrip): Observable<unknown> {
```

Comment:
You should know the type that is being returned. Even if it's more than one type possible. In this case i'm not entirely sure what you want to return when there is an error. Maybe you want to return an empty ITrip. Maybe you want to return null or undefined. You do need to make sure you return something from catchError and indicate the type the calling code can expect.